### PR TITLE
Improved initial loading screen on web and native client

### PR DESCRIPTION
**Pull request by stijuh**

### CODE REVIEW

File: GainsTracker.ClientNative/wwwroot/index.html [hunk 1]
```diff
@@ -5,23 +5,73 @@
...
+            align-items: center;
+            height: 100vh;
+        }
+
+        .loading {
+            width: 5rem;
+            height: 5rem;
+            border-radius: 50%;
+            background-color: #333;
+            margin: 0 5px;
```
Comment (Junmii) [inline GainsTracker.ClientNative/wwwroot/index.html:34]:
```suggestion
```

Remove the margin and add a gap instead!

File: GainsTracker.ClientNative/wwwroot/index.html [hunk 2]
```diff
@@ -5,23 +5,73 @@
...
     <link href="https://cdnjs.cloudflare.com/ajax/libs/animate.css/4.0.0/animate.min.css" rel="stylesheet"/>
+    
     <script src="_content/GainsTracker.UI/script/ElementManipulator.js" type="text/javascript"></script>
+    
+    <style>
+        .loading-container {
+            display: flex;
+            justify-content: center;
+            align-items: center;
+            height: 100vh;
```
Comment (Junmii) [inline GainsTracker.ClientNative/wwwroot/index.html:26]:
```suggestion
            height: 100vh;
            gap: 0.5rem;
```
Work with just rem or px for consistency (and rem is more accessibility approved! (if i remember correctly???))

File: GainsTracker.ClientWebAssembly/wwwroot/index.html [hunk 3]
```diff
@@ -1,30 +1,70 @@
...
+            height: 100vh;
+            background-color: var(--primary-color);
+        }
+
+        .loading {
+            width: 5rem;
+            height: 5rem;
+            border-radius: 50%;
+            background-color: #333;
+            margin: 0 5px;
```
Comment (Junmii) [inline GainsTracker.ClientWebAssembly/wwwroot/index.html:32]: Same comments here as upstairs

File: GainsTracker.ClientWebAssembly/wwwroot/index.html [hunk 4]
```diff
@@ -1,30 +1,70 @@
...
+            <div class="loading"></div>
+            <div class="loading"></div>
+            <div class="loading"></div>
+        </div>
+    </div>
+    
+    <div id="blazor-error-ui">
+        An unhandled error has occurred.
+        <a href="" class="reload">Reload</a>
+        <a class="dismiss">🗙</a>
```
Comment (Junmii) [inline GainsTracker.ClientWebAssembly/wwwroot/index.html:64]: This bad boy seems to not function as an <a>

**Review by Junmii — APPROVED** [review]
Looks very nice overall!! Gud 
SmoothMoveCowGIF

File: GainsTracker.ClientWebAssembly/wwwroot/index.html (hunk 3 above)
Comment (stijuh) [reply GainsTracker.ClientWebAssembly/wwwroot/index.html:32]: Yes ma'am

**Pull request merged**